src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    protected Object clone() throws CloneNotSupportedException {
+        return super.clone();
+    }
```

Comment:
same

Reply:
It was auto generated.
It isn't necessarily method.
I deleted it.